src/eventProcessor/EventProcessorInterface.js

```diff
@@ -39,7 +41,4 @@ class EventProcessor {
    *     If Number.POSITIVE_INFINITY, data will be stored forever.
    */
   persistTime(key, value) {};
-
 }
-
-exports = EventProcessor;
```

Comment:
I think not exporting the types might cause a problem, since IIRC the concrete implementations will need to require them (or at least goog.requireType them) for @interface JSDoc annotations.

That said, I think the API looks good so I'm going to merge this.  Feel free to look into this and address it in a followup PR if necessary.